Autocoders/MagicDrawCompPlugin/README

```diff
@@ -43,13 +43,13 @@ To regenerate the Documentation to conform with any future changes:
 - In the 'File' tab at the top of the Doxygen GUI, choose open and select the file titled "Doxyfile"
 - This will load all the configurations necessary to regenerate the documentation 
 - Go to the 'Run' tab in the GUI and click the 'Run Doxygen' button to regenerate the documentation
-- To regenerate the refman.pdf file froun in the doc folder, open doc/rtf/refman.rtf, save as a Word Document, convert the document to PDF, then drag the PDF to the doc folder
+- To regenerate the refman.pdf file found in the doc folder, open doc/rtf/refman.rtf, save as a Word Document, convert the document to PDF, then drag the PDF to the doc folder
 - To edit the comments found in the top of the documentation files, edit the comment block found at the top of src/gov/nasa/jpl/componentaction/MainMenuConfigurator.java
 
 
 Version 2.7:
 -------------
-- Add the subsystem name as a field in the <assmbly> tag.  This allows the Autocoder to generate a unique name for each topology model.
+- Add the subsystem name as a field in the <assembly> tag.  This allows the Autocoder to generate a unique name for each topology model.
```

Comment:
This appears to be a changelog. Some projects aren't interested in correcting historical documents. I can trivially exclude files if that's your approach.